README.md

```diff
@@ -4,9 +4,8 @@ https://github.com/rjl20/hpmor
 Maintainer: Reuben Thomas <rrt@sc3d.org>
 
 A LaTeX version of [the popular didactic fan-fiction](http://www.hpmor.com)
-by Eliezer Yudkowsky, which can make a PDF e-book (one file) or printable
-books (either one or six volumes; the latter option is more practical to
-bind). There are also dust jackets for the printable volumes.
+by Eliezer Yudkowsky, which can make e-books in PDF, ePub and Mobi formats,
+and six PDF volumes that can be printed and bound. There are also dust jackets for the printable volumes.
```

Comment:
This line still needs re-wrapping! Sorry, I still can't push to this branch for some reason.

Reply:
(I think you applied my patch immediately, so didn't see that I updated it.)

Reply:
yes, sorry, I was too lazy to google how to handle our fix, so I just copied the text.

Reply:
No worries. This line still needs rewrapping!

Reply:
now than line should be fine